### PR TITLE
Add extraction support for png, jpeg, tiff, and svg; and VLM captioning stage

### LINKED ISSUE #191: [FEA]: Add support for table and chart extraction from images, docx files, and pptx files.
### Is this a new feature, an improvement, or a change to existing functionality?

New Feature

### How would you describe the priority of this feature request

Currently preventing usage

### Please provide a clear description of problem this feature solves

Currently the processing pathways for images/docx/pptx files do not integrate image extraction components for tables and charts. This issue will introduce support for those features.

### Describe the feature, and optionally a solution or implementation and any alternatives

New functionality will include updates to `docx_helper.py`, `pptx_helper.py`, to include table and chart detection via yolox, and downstream table/chart extractors; it will also add an extraction stage for jpg, jpeg, png, and svg elements that leverages yolox and downstream table/chart extractors.

### Additional context

_No response_

### LINKED ISSUE #207: [FEA]: Add Image Extraction Stage to Pipeline
### Is this a new feature, an improvement, or a change to existing functionality?

New Feature

### How would you describe the priority of this feature request

Currently preventing usage

### Please provide a clear description of problem this feature solves

New stage that will handle svg, png, jpeg, and jpg

### Describe the feature, and optionally a solution or implementation and any alternatives

Adds a pipeline stage to process image files

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by drobison00**
Resolves #191
Resolves #207 

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/nv-ingest/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

File: src/nv_ingest/extraction_workflows/image/image_handlers.py [hunk 1]
```diff
@@ -0,0 +1,447 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
```
Comment (edknv) [inline src/nv_ingest/extraction_workflows/image/image_handlers.py:5]: Move this down after the licnese text?

File: src/nv_ingest/schemas/metadata_schema.py [hunk 2]
```diff
@@ -317,6 +311,7 @@ class InfoMessageMetadataSchema(BaseModelNoExt):
 # Main metadata schema
 class MetadataSchema(BaseModelNoExt):
     content: str = ""
+    content_url: str = ""
```
Comment (edknv) [inline src/nv_ingest/schemas/metadata_schema.py:314]: I don't see this used anywhere. How would this be used?

File: src/nv_ingest/util/pipeline/stage_builders.py [hunk 3]
```diff
@@ -319,14 +343,26 @@ def add_nemo_splitter_stage(pipe, morpheus_pipeline_config, ingest_config):
...
+    auth_token = os.environ.get(
+        "NVIDIA_BUILD_API_KEY",
+        "",
+    ) or os.environ.get(
+        "NGC_API_KEY",
+        "",
+    )
+
+    endpoint_url = os.environ.get("NVIDIA_BUILD_CAPTION_ENDPOINT",
+                                  "https://ai.api.nvidia.com/v1/gr/meta/llama-3.2-90b-vision-instruct/chat/completions")
```
Comment (edknv) [inline src/nv_ingest/util/pipeline/stage_builders.py]: Do you think we should also add this to docker-compose.yaml?

**Review by edknv — APPROVED** [review]

File: src/nv_ingest/schemas/metadata_schema.py (hunk 2 above)
Comment (drobison00) [reply src/nv_ingest/schemas/metadata_schema.py:314]: Its pulled in from another PR that provides a url reference to the data. The use case now is the "store_to_file" flag in the CLI that strips out content and stores it in a local file.

File: src/nv_ingest/util/pipeline/stage_builders.py (hunk 3 above)
Comment (drobison00) [reply src/nv_ingest/util/pipeline/stage_builders.py]: Yeah, that probably makes sense.

**Pull request merged**